backend/dataall/modules/mlstudio/cdk/mlstudio_stack.py

```diff
@@ -38,6 +38,11 @@ class SageMakerDomainExtension(EnvironmentStackExtension):
     @staticmethod
     def extent(setup: EnvironmentSetup):
         _environment = setup.environment()
+        with setup.get_engine().scoped_session() as session:
```

Comment:
Good catch! thank you!

Reply:
Maybe as a future improvement idea: could we use Enums or other type for the env parameter values? Right now we need to look for the particular string in the code, both in frontend and backend

Reply:
Yes, we should do something about that)